src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -78,17 +77,26 @@ cdef class _AioCall:
         """Destroys the corresponding Core object for this RPC."""
         grpc_call_unref(self._grpc_call_wrapper.call)
 
-    cdef AioRpcStatus _cancel_and_create_status(self, object cancellation_future):
-        """Cancels the RPC in Core, and return the final RPC status."""
-        cdef AioRpcStatus status
+    @property
+    def locally_cancelled(self):
```

Comment:
IMO this method would need to be exposed in some way at `Call` and `RpcError` level, for doing something like this:

```python
try:
    await call
except RpcError as err:
    if err.code() == CANCELLED and not err.locally_cancelled():
        # Retry!
```

Reply:
I would like to make semantic as simple as possible, users or us should not worry about whether an RPC is cancelled locally or remotely. Sorry, I wasn't clear about that design before. I'm open to expose the `_is_locally_cancelled` flag, but in a way that is less invasive.

The goal of the `_is_locally_cancelled` flag is only be used in order to free resources. For cancellation detection, the final status code is the ground truth.

So, I temporarily removed this flag until there is a valid use case for it.

Reply:
Ok let's find the right example for implementing it.